Controllers/UserController.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Controllers;
+
+use Models\UserModel;
+
+class UserController implements IController
+{
+    private $model;
+    private $fc;
+
+    public function __construct()
+    {
+        $this->model = new UserModel();
+        $this->fc = FrontController::getInstance();
+    }
+
+    public function createUserAction()
+    {
+        $imgName = "";
```

Comment:
Don't need to use double quotes here. Specify such a variables using single one.
`$someVar = '';`